### ISSUE TITLE
Call to undefined method PhpParser\Node\Expr\ArrayDimFetch::toString()

### ISSUE THREAD

**Issue by theNailz** [monque/PHP-Migration#13]
```
php phpmig.phar app/
```

```
PHP Fatal error:  Call to undefined method PhpParser\Node\Expr\ArrayDimFetch::toString() in phar://C:/dev/inos/api/phpmig.phar/src/Changes/v5dot3/IncompByReference.php on line 396
PHP Stack trace:
PHP   1. {main}() C:\dev\inos\api\phpmig.phar:0
PHP   2. PhpMigration\App->run() C:\dev\inos\api\phpmig.phar:9
PHP   3. PhpMigration\App->commandMain() phar://C:/dev/inos/api/phpmig.phar/src/App.php:194
PHP   4. PhpParser\NodeTraverser->traverse() phar://C:/dev/inos/api/phpmig.phar/src/App.php:399
PHP   5. PhpParser\NodeTraverser->traverseArray() phar://C:/dev/inos/api/phpmig.phar/vendor/nikic/php-parser/lib/PhpParser/NodeTraverser.php:64
PHP   6. PhpParser\NodeTraverser->traverseNode() phar://C:/dev/inos/api/phpmig.phar/vendor/nikic/php-parser/lib/PhpParser/NodeTraverser.php:135
PHP   7. PhpParser\NodeTraverser->traverseArray() phar://C:/dev/inos/api/phpmig.phar/vendor/nikic/php-parser/lib/PhpParser/NodeTraverser.php:84
PHP   8. PhpParser\NodeTraverser->traverseNode() phar://C:/dev/inos/api/phpmig.phar/vendor/nikic/php-parser/lib/PhpParser/NodeTraverser.php:135
PHP   9. PhpParser\NodeTraverser->traverseArray() phar://C:/dev/inos/api/phpmig.phar/vendor/nikic/php-parser/lib/PhpParser/NodeTraverser.php:84
PHP  10. PhpParser\NodeTraverser->traverseNode() phar://C:/dev/inos/api/phpmig.phar/vendor/nikic/php-parser/lib/PhpParser/NodeTraverser.php:135
PHP  11. PhpParser\NodeTraverser->traverseArray() phar://C:/dev/inos/api/phpmig.phar/vendor/nikic/php-parser/lib/PhpParser/NodeTraverser.php:84
PHP  12. PhpParser\NodeTraverser->traverseNode() phar://C:/dev/inos/api/phpmig.phar/vendor/nikic/php-parser/lib/PhpParser/NodeTraverser.php:135
PHP  13. PhpParser\NodeTraverser->traverseArray() phar://C:/dev/inos/api/phpmig.phar/vendor/nikic/php-parser/lib/PhpParser/NodeTraverser.php:84
PHP  14. PhpParser\NodeTraverser->traverseNode() phar://C:/dev/inos/api/phpmig.phar/vendor/nikic/php-parser/lib/PhpParser/NodeTraverser.php:135
PHP  15. PhpMigration\CheckVisitor->leaveNode() phar://C:/dev/inos/api/phpmig.phar/vendor/nikic/php-parser/lib/PhpParser/NodeTraverser.php:101
PHP  16. PhpMigration\Changes\v5dot3\IncompByReference->leaveNode() phar://C:/dev/inos/api/phpmig.phar/src/CheckVisitor.php:176
PHP  17. PhpMigration\Changes\v5dot3\IncompByReference->populateCall() phar://C:/dev/inos/api/phpmig.phar/src/Changes/v5dot3/IncompByReference.php:254

Fatal error: Call to undefined method PhpParser\Node\Expr\ArrayDimFetch::toString() in phar://C:/dev/inos/api/phpmig.phar/src/Changes/v5dot3/IncompByReference.php on line 396

Call Stack:
    0.0034     233144   1. {main}() C:\dev\inos\api\phpmig.phar:0
    0.0047     367168   2. PhpMigration\App->run() C:\dev\inos\api\phpmig.phar:9
    0.0047     369000   3. PhpMigration\App->commandMain() phar://C:/dev/inos/api/phpmig.phar/src/App.php:194
    4.9199    6224744   4. PhpParser\NodeTraverser->traverse() phar://C:/dev/inos/api/phpmig.phar/src/App.php:399
    4.9200    6195928   5. PhpParser\NodeTraverser->traverseArray() phar://C:/dev/inos/api/phpmig.phar/vendor/nikic/php-parser/lib/PhpParser/NodeTraverser.php:64
    4.9201    6196776   6. PhpParser\NodeTraverser->traverseNode() phar://C:/dev/inos/api/phpmig.phar/vendor/nikic/php-parser/lib/PhpParser/NodeTraverser.php:135
    4.9205    6197712   7. PhpParser\NodeTraverser->traverseArray() phar://C:/dev/inos/api/phpmig.phar/vendor/nikic/php-parser/lib/PhpParser/NodeTraverser.php:84
    4.9207    6152120   8. PhpParser\NodeTraverser->traverseNode() phar://C:/dev/inos/api/phpmig.phar/vendor/nikic/php-parser/lib/PhpParser/NodeTraverser.php:135
    4.9207    6152992   9. PhpParser\NodeTraverser->traverseArray() phar://C:/dev/inos/api/phpmig.phar/vendor/nikic/php-parser/lib/PhpParser/NodeTraverser.php:84
    4.9228    6148072  10. PhpParser\NodeTraverser->traverseNode() phar://C:/dev/inos/api/phpmig.phar/vendor/nikic/php-parser/lib/PhpParser/NodeTraverser.php:135
    4.9231    6149312  11. PhpParser\NodeTraverser->traverseArray() phar://C:/dev/inos/api/phpmig.phar/vendor/nikic/php-parser/lib/PhpParser/NodeTraverser.php:84
    4.9336    6157224  12. PhpParser\NodeTraverser->traverseNode() phar://C:/dev/inos/api/phpmig.phar/vendor/nikic/php-parser/lib/PhpParser/NodeTraverser.php:135
    4.9349    6159064  13. PhpParser\NodeTraverser->traverseArray() phar://C:/dev/inos/api/phpmig.phar/vendor/nikic/php-parser/lib/PhpParser/NodeTraverser.php:84
    4.9374    6161328  14. PhpParser\NodeTraverser->traverseNode() phar://C:/dev/inos/api/phpmig.phar/vendor/nikic/php-parser/lib/PhpParser/NodeTraverser.php:135
    4.9416    6165456  15. PhpMigration\CheckVisitor->leaveNode() phar://C:/dev/inos/api/phpmig.phar/vendor/nikic/php-parser/lib/PhpParser/NodeTraverser.php:101
    4.9417    6167736  16. PhpMigration\Changes\v5dot3\IncompByReference->leaveNode() phar://C:/dev/inos/api/phpmig.phar/src/CheckVisitor.php:176
    4.9417    6167784  17. PhpMigration\Changes\v5dot3\IncompByReference->populateCall() phar://C:/dev/inos/api/phpmig.phar/src/Changes/v5dot3/IncompByReference.php:254```
```

php -v
```
php -v
PHP 5.6.19 (cli) (built: Mar  2 2016 20:08:35)
Copyright (c) 1997-2016 The PHP Group
Zend Engine v2.6.0, Copyright (c) 1998-2016 Zend Technologies
    with Xdebug v2.2.5, Copyright (c) 2002-2014, by Derick Rethans
```

`app/` contains our Laravel 5.2 project, which I am unable to share, although I could provide more pointers or other info if requested.

**Comment by monque** [comment]
Could you re-run this command with `-v`? It will output like this

```
[INFO] Parse file ./src/resources/views/welcome.blade.php
[INFO] Parse file ./src/server.php
[INFO] Parse file ./src/tests/ExampleTest.php
[INFO] Parse file ./src/tests/TestCase.php
```

Then paste the `static function call` in the last file, it seems that some "dynamic" call cause this bug.

**Comment by monque** [comment]
Fixed, should try [v0.2.2](https://github.com/monque/PHP-Migration/releases/tag/v0.2.2)
